airflow/configuration.py

```diff
@@ -1249,7 +1251,7 @@ def read_dict(  # type: ignore[override]
         """
         super().read_dict(dictionary=dictionary, source=source)
 
-    def has_option(self, section: str, option: str, lookup_from_deprecated_options: bool = True) -> bool:
+    def has_option(self, section: str, option: str, lookup_from_deprecated: bool = True) -> bool:
```

Comment:
Could you please add the parameter (`lookup_from_deprecated`) to the method doc as well? 
`:param lookup_from_deprecated: ...`
It is already missing but since the name is also changing maybe, good to include a description here of what that parameter is controlling and when to pass it.